taca/element/Element_Runs.py

```diff
@@ -379,44 +374,38 @@ def find_manifest_zip(self):
             logger.warning(
                 f"No manifest found for run '{self.run_dir}' with pattern '{glob_pattern}'."
             )
-            return False  # TODO: determine whether to raise an error here instead
+            return None
         elif len(glob_results) > 1:
             logger.warning(
                 f"Multiple manifests found for run '{self.run_dir}' with pattern '{glob_pattern}', using latest one."
             )
             glob_results.sort()
```

Comment:
I guess this works as long as no-one renames the manifest files, which we do for illumina during redemux. Fine for now but we need to think about it later.

Reply:
If redemux is done manually, how about overriding the demux file path as a CLI parameter?

Reply:
Yeah I think that would make sense 👍